Tests/TuistDependenciesTests/SwiftPackageManager/Utils/PackageInfoMapperTests.swift

```diff
@@ -1562,7 +1602,7 @@ extension PackageInfoMapping {
         basePath: AbsolutePath = "/",
         packageInfos: [String: PackageInfo] = [:],
         platforms: Set<TuistGraph.Platform> = [.iOS],
-        targetDependencyToFramework: [String: Path] = [:],
+        targetDependencyToFramework _: [String: Path] = [:],
```

Comment:
this was done by `./fourier format swift --fix`

Reply:
Then it means it's not used, you can remove the parameter